.github/workflows/static_checks.yaml

```diff
@@ -0,0 +1,24 @@
+name: Static Checks
+on:
+    pull_request: {}
+    push: {}
+jobs:
+    go_lint:
+        name: Go Lint
+        runs-on: ubuntu-18.04
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@master
+            - name: Run linter
+              uses: golangci/golangci-lint-action@v1
+              with:
+                  version: v1.29
+    go_format:
```

Comment:
Can you confirm if `fmt` is not part of the `golangci-lint-action` so that we can remove this if its already there. Also can you remove the `./bin/goimports` as we are not using that anymore?

Reply:
Yep! `fmt` can be enabled in the CI using a flag. I've made the required changes and cleaned up the `bin/` folder. 😄